rosdep/python.yaml

```diff
@@ -2812,6 +2812,9 @@ python-qt5-bindings-gl:
     xenial: [python-pyqt5.qtopengl]
     yakkety: [python-pyqt5.qtopengl]
     zesty: [python-pyqt5.qtopengl]
+python-qt5-bindings-qsci:
+  ubuntu:
+    xenial: [python-pyqt5.qsci]
```

Comment:
This looks to be availabe on xenial and newer: https://packages.ubuntu.com/xenial/amd64/python-pyqt5.qsci I'd rather see a non-versioned rule for it so we don't have to updated it in the future, and it will work for newer platforms too.

Reply:
It's also available on stretch and newer: https://packages.debian.org/sid/python-pyqt5.qsci A key for debian would be good.

Reply:
The generic fedora and gentoo rules could also be copied in here.